### ISSUE TITLE
[Proposal] Нерф грызунов и их способности жратъ проводку

### ISSUE THREAD

**Issue by WatlerJe** [TauCetiStation/TauCetiClassic#9416]
#### Подробное описание проблемы
Ах-ох приятное дерьмо. Крутая механика с грузынами которые не просто спавнятся на карте, а еще и портят проводку тем самым подкидывая работу инженерам. Круто классно, но есть один нюанс. Грызуны обладают лютейшим сопротивлением к току и способны выдержать любую силу в сети, что дает им возможность ЖРАТЪ медный кабель десятками метров.

Мое предложение по их нерфу:
- При попытке напасть на кабель ящерка или мышка дико ликвидируются об него;
- Вместо исчезновения кабеля в никуда, на месте обрыва спавнится 2 куска кабеля (столько же спавнится если обрезать кабель кусачками);

Таким образом грызуны по прежнему останутся угрозой для станции, но не будут сжирать чрезмерное количество кабеля.

**Comment by Deahaka** [comment]
Пусть грызут, они ж для этого и научены были.

**Comment by Pivo24** [comment]
Научены Синдикатом

:joy:

**Comment by KIBORG04** [comment]
Я согласен, плохо, что кабель просто куделится, плохо, что можно за пару секунд выжрать теха

**Comment by Deahaka** [comment]
> Я согласен, плохо, что кабель просто куделится, плохо, что можно за пару секунд выжрать теха

И за эти же пару секунд провести назад проводки... А можно просто убить вредителей сразу.
